src/lib/tabs/tabs.md

```diff
@@ -80,6 +80,30 @@ The `tab-nav-bar` is not tied to any particular router; it works with normal `<a
 the `active` property to determine which tab is currently active. The corresponding
 `<router-outlet>` can be placed anywhere in the view.
 
+## Lazy Loading
+By default, the tab contents are eagerly loaded. Eagerly loaded tabs
```

Comment:
Should we mention here about how lazy-loading resolves issues with components that expect to be in the DOM while they setup? e.g. expansion panels and nested ink bars.

I suspect we'll want a place to point people to as we continue to get issues about this

Reply:
Preferably yes. I'm waiting on a fix to https://github.com/angular/material2/issues/5269, to render expansion panels within tabs....as are others.

I'll probably use lazy loading in combination with a the [Progress Spinner](https://material.angular.io/components/progress-spinner/overview) component since I render many expansion panels within a tab (see EDIT view on [JSONSchema.Net](JSONSchema.Net)).

Eagerly awaiting this feature :)

Reply:
@andrewseguin - Added, let me know what u think?